### ISSUE TITLE
[Feature Request]: Protally like indicators

### ISSUE THREAD

**Issue by Keksstar** [josephdadams/TallyArbiter#461]
### What feature would you like to see added to TallyArbiter?

Would be nice, if an module can be build to work like Protally to show onscreen-tally on computers. Sadly Protally has large graphics bugs, so is unable to show borders.

**Comment by josephdadams** [comment]
Yes, this would be great. I was thinking I had already added this as a feature request but can’t find it.

**Comment by JTF4** [comment]
That would be cool! It would probably be best implemented as a listener instead of a server in TA.

One thing that we could consider adding to the server would be the ability to open a embedded window that just shows the tally state.

**Comment by josephdadams** [comment]
Yeah a listener client for sure.
